### PR TITLE
Fix watchable tests

### PR DESCRIPTION
**Pull request by petern-sc**
Was trying to make another change but noticed the tests were always green.

Currently the watchable tests don't run because `(IO[Unit], IO[Unit]).parSequence => IO[(IO[Unit], Unit)]`. Change to List to parSequence works as intended

Correct watch events logic to make the tests pass

### CODE REVIEW

**Comment by petern-sc** [comment]
@joan38 don't seem to be able to request review via the github UI, if you get a chance 🙏

**Comment by remibesse** [comment]
I'll be looking into this. Thanks @petern-sc

File: kubernetes-client/test/src/com/goyeau/kubernetes/client/operation/WatchableTests.scala [hunk 1]
```diff
@@ -132,7 +140,7 @@ trait WatchableTests[F[_], Resource <: { def metadata: Option[ObjectMeta] }]
         else
           Map(defaultNamespace -> expectedEvents)
 
-      (
+      List(
         watchEvents(expected, name, None),
         F.sleep(100.millis) *> sendEvents(defaultNamespace, name) *> sendToAnotherNamespace(name)
       ).parSequence
```
Comment (joan38) [inline kubernetes-client/test/src/com/goyeau/kubernetes/client/operation/WatchableTests.scala]: I think that would be better if it stays as a tuple and that `parSequence` needs to be `parTupled`.

File: kubernetes-client/test/src/com/goyeau/kubernetes/client/operation/WatchableTests.scala [hunk 2]
```diff
@@ -145,7 +153,7 @@ trait WatchableTests[F[_], Resource <: { def metadata: Option[ObjectMeta] }]
       val name     = s"${resourceName.toLowerCase}-watch-single"
       val expected = Set[EventType](EventType.ADDED, EventType.MODIFIED, EventType.DELETED)
 
-      (
+      List(
         watchEvents(Map(defaultNamespace -> expected), name, Some(defaultNamespace)),
         F.sleep(100.millis) *> sendEvents(defaultNamespace, name) *> sendToAnotherNamespace(name)
       ).parSequence
```
Comment (joan38) [inline kubernetes-client/test/src/com/goyeau/kubernetes/client/operation/WatchableTests.scala]: Same as above:
I think that would be better if it stays as a tuple and that `parSequence` needs to be `parTupled`.

**Review by joan38 — CHANGES_REQUESTED** [review]

**Pull request merged**